src/blockchain/src/block.rs

```diff
@@ -75,7 +79,16 @@ impl Block {
     }
 }
 
-// struct Transaction define the details of a transaction in a block
+/// ## Transactions ##
```

Comment:
Did you check the rendered output... see's odd to repeat the name

bad markdown

```suggestion
/// ## Transaction
```